docs/js/doc.js

```diff
@@ -84,7 +84,7 @@
         source += '  ';
       }
       source += line + '\n';
-      if (line.indexOf('<') === 0 && line.indexOf('</') !== 0) {
+      if (line.indexOf('<') === 0 && line.indexOf('</') !== 0 && line.indexOf('<input') === -1) {
```

Comment:
It indents if the line is starting with a tag, but NOT a closing tag, and NOT a one-line tag (only `<input>` here) :)